ansible/playbooks/paas/main.yml

```diff
@@ -110,22 +110,20 @@
   become: true
   vars:
     certificate_ca_host: "localhost"
-    certificate_ca_host_dir: "tls/ca"
+    certificate_ca_host_dir: "../../tls"
```

Comment:
Since you added this feature in your previous commit, this change should have been included as well. ;-)